packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -228,12 +229,22 @@ export interface ISummaryConfigurationHeuristics extends ISummaryBaseConfigurati
      * Defines the maximum number of Ops, since the last received Ack, that can be allowed
      * before running the summary with reason maxOps.
      */
-     maxOps: number;
+    maxOps: number;
     /**
      * Defnines the minimum number of Ops, since the last received Ack, that can be allowed
      * before running the last summary.
      */
     minOpsForLastSummaryAttempt: number;
+    /**
+     * Defines the minimum for the allowed time in between summarizations.
+     * Pairs with maxIdleTime to form a range.
+     */
+    minIdleTime: number;
+    /**
+     * Defines the maximum for the allowed time in between summarizations.
```

Comment:
Its unclear from the documentation how this relates to maxTime.

What does "Idle" mean here? (the docs should make this clear)

Also, we allow unlimited time between summarizations, right? (The documentation as is disagrees with this). I think our time limits are about how long, when we have an active client that can summarize running, we will allow an acked op to exist without starting a summary that will include it. This is distinct from the time between summaries.

My guess about how these 3 times relate (but the docs as they are don't imply this):

After receiving an ack for an op, we wait minIdleTime. If it is not included in a summary already, wait until idle or maxTime, which ever comes first, then perform a summary.

This only involves 2 of the 3 times, so I'm confused.

Reply:
`maxTime` refers to the maximum amount of time in which we are receiving ops but have not yet run a summary. For example, say we receive ops one by one just before the idle time is triggered. In this case, we still want to run the summary since it's been a while since the last one.

`minIdleTime` and `maxIdleTime` work together to form the idle timer and they essentially work as upper and lower boundaries in our calculation on the true idle time to be used. For example, if we only receive 1 op, we don't want to have the same idle time as say 100 ops. Based on the boundaries we set in those properties, the idle time will change linearly depending on the number of ops we receive.

Reply:
That is more clear. You should but that improved explanation in the doc comments.